pages/api/graphql.ts

```diff
@@ -7,6 +8,11 @@ import typeDefs from '../../graphql/typeDefs'
 import resolvers from '../../graphql/resolvers'
 import { PrismaSessionStore } from '@quixo3/prisma-session-store'
 import { prisma } from '../../prisma'
+import { apolloLogPlugin } from '../../helpers/apolloLogPlugin'
+
+// Temporary fix to broken sentry package.
```

Comment:
Since this is temporary, can you write a `TODO: ` comment as well? For example `TODO: remove import when permanent fixed is implemented`

Reply:
added the todo comment, thanks.